app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :votes
+  acts_as_voter
+  validates :name, presence: true, exclusion: { in: ['', nil] }
```

Comment:
What does the `exclusion: { ... }` part do? Why do we need it or want it? How is it difference from `presence: true`?